articles/quickstart/config.yml

```diff
@@ -1,4 +0,0 @@
-author:
```

Comment:
What about those articles without a specific author entry. This was here to show a default in that case. Can we use something like "Auth0" or "SDKs Team" and have it display an auth0 avatar? 🤔

Reply:
@lbalmaceda Yeah, agreed. Should we provide and email? If the answer is "yes", which one?

Reply:
@davidpatrick Is an email required for the fallback? Maybe to load the gravatar?

Reply:
Yes, I think we should because of gravatar. But no clue which one. I know `support@auth0.com` has a valid gravatar image associated (with the Auth0 logo). Maybe that's enough?

Reply:
@davidpatrick Is the email field actually required? What happens if it's not provided.

Reply:
The author isn't necessarily required - https://github.com/auth0/auth0-docs/blob/master/test/schemas/author.js

But I believe it is assumed one is there https://github.com/auth0/auth0-docs/blob/master/client/components/quickstarts/Author/index.js#L27

Not sure what happens when one isn't supplied 😅.  Could test in a review app, or we could just use `support@auth0.com` as @lbalmaceda has mentioned

Reply:
OK, let's use the support address then @alexisluque. Thanks @davidpatrick!

Reply:
Would be best to change this file to https://github.com/auth0/docs/pull/8033

Reply:
Please rebase @alexisluque . This file should remain, as we recently merged this https://github.com/auth0/docs/pull/8033